docusaurus/docs/testing/e2e-test.md

```diff
@@ -4,6 +4,22 @@ This repo is configured for end-to-end testing with [Cypress](https://docs.cypre
 
 Because of this, we extend the [Cypress best practices](https://docs.cypress.io/guides/references/best-practices#How-It-Works), so be sure to read them before write any test.
 
+## Quick start
+
+> This covers running E2E tests locally in a vanilla environment
+
+### Pre-requisites
+- Instance of Rancher deployed and set up (passed setup pages)
+- Dashboard running locally at the default address (`https://localhost:8005`) pointing to the set up rancher
+
+### Run the tests
+
+This will start the cypress test runner, where you can select which tests to run
+
+```
+TEST_PASSWORD=<rancher admin password> TEST_SKIP_SETUP=true yarn cy:open
```

Comment:
I do not know if you changed anything lately, but theoretically you need also `TEST_USERNAME ` and `TEST_BASE_URL`.
`TEST_BASE_URL` points to default to the URL you mentioned.
If it works anyway without username, then just set this resolved.

Not so sure why we prefer inlined env vars to a config as standard workspace, but at least like this is clear also for who does not know what are them.

Reply:
At some point `cypress.config.ts` was updated to have defaults / fall back's for both.

I used inlined env vars here just for simplicity. Your docs below are good for anyone who needs to dive deeper or do anything other than basic tests